### PR TITLE
Update name

### LINKED ISSUE #110: [Account] Allow user to modify his name

### PR DESCRIPTION
**Pull request by Alketta**
Close #110

### CODE REVIEW

File: front/src/locales/fr.json [hunk 1]
```diff
@@ -1085,6 +1084,17 @@
       "right": "Droite",
       "center": "Centré",
       "justify": "Justifié"
+    },
+     "roles": {
+      "OWNER": "Propriétaire",
+      "ADMIN": "Admin",
```
Comment (mdartic) [inline front/src/locales/fr.json:1090]: Do you think "Administrateur" could be a better translation ?
Idem for profiles ?

Comment (Alketta) [reply front/src/locales/fr.json:1090]: I think we are all used to seeing/using Admin.

Comment (mdartic) [reply front/src/locales/fr.json:1090]: Mmm... well, I do not agree, sometimes it's translated as "Administrateur" entirely (eg, on Linux Mint OS). 
But, ok, we'll see later if people don't understand.

**Comment by mdartic** [comment]
Could you fix Sonar errors ? Your `<i>` tags are not ok, prefer using `<em>`, and there are 2 bugs easy to fix too.

**Review by mdartic — APPROVED** [review]

**Pull request merged**